### PR TITLE
Fixups

### PR DESCRIPTION
**Pull request by bakerboy448**
various fixups; see commits

### CODE REVIEW

**Comment by Taloth** [comment]
That reformatting is horrible, sry but that's totally not what I want to see.

**Comment by bakerboy448** [comment]
fair enough; blame vscode and markdown prettifier; 

backing it out momentarily

File: src/sections/downloads-v3/windows.marko [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
+    <p>Sonarr is supported natively on Windows. Sonarr can be installed on Windows as Windows Service or system tray application.</p>
+    <p>
+      A Windows Service runs even when the user is not logged in, but special care must be taken since Windows Services cannot access network drives (X:\ mapped drives) without special configuration steps.<br/>
+      Additionally the Windows Service runs under the 'Local Service' account, by default this account does not have permissions to access your user's home directory unless permissions have been assigned manually. This is particularly relevant when using download clients that are configured to download to your home directory.<br/>
+      'Local Service' also generally has very limited permissions. It's therefore advisable to install Sonarr as a system tray application if the user can remain logged in. The option to do so is provided during the installer.
+    </p>
+    <alert level="warning">
+      Windows versions are limited for support to those currently supported by Microsoft, others may work but this is an unsupported configuration.
```
Comment (Taloth) [inline src/sections/downloads-v3/windows.marko]: Why change this? To cover older Windows Server versions? Tbh I do like the explicit mention of XP & 7.

File: src/sections/downloads-v3/windows.marko [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-            Additionally the Windows Service runs under the 'Local Service' account, by default this account does not have permissions to access your user's home directory unless permissions have been assigned manually. This is particularly relevant when using download clients that are configured to download to your home directory.<br/>
-            It's therefore advisable to install Sonarr as a system tray application if the user can remain logged in. The option to do so is provided during the installer.
-        </p>
-        <alert level="warning">Sonarr v3 no longer supports Windows XP. At this time Windows 7 <i>should</i> still work, but we will not maintain Windows 7 support.</alert>
+    <p>Sonarr is supported natively on Windows. Sonarr can be installed on Windows as Windows Service or system tray application.</p>
+    <p>
+      A Windows Service runs even when the user is not logged in, but special care must be taken since Windows Services cannot access network drives (X:\ mapped drives) without special configuration steps.<br/>
```
Comment (Taloth) [inline src/sections/downloads-v3/windows.marko]: No `\\server\share` isn't useable by windows services either if it runs as Local Service or even System due to potential permissions "without special configuration steps"

File: src/sections/support.marko [hunk 3]
```diff
@@ -9,12 +9,12 @@
         </div>
     </a>
 
-    <a class="col-sm-3" href="https://wiki.sonarr.tv" target="_blank">
+    <a class="col-sm-3" href="https://wiki.servarr.com/sonarr" target="_blank">
```
Comment (Taloth) [inline src/sections/support.marko]: Doesn't https://wiki.sonarr.tv already redirect to servarr?

File: src/sections/downloads-v3/docker.marko [hunk 4]
```diff
@@ -1,62 +1,62 @@
...
-                <p>
-                    <a href="https://hotio.dev">hotio</a> doesn't specify any default volumes, besides <code>/config</code>. Images are automatically updated 2x in 1 hour if upstream changes are found.
-                    Read the <a href="https://hotio.dev/containers/sonarr">instructions</a> on how to install the image.
-                </p>
-            </li>
-            </ul>
-        </p>
-    </install-step>
-</dl>
+  <install-step title="Introduction">
```
Comment (Taloth) [inline src/sections/downloads-v3/docker.marko]: Why drop the maintained-by?

File: src/sections/downloads-v3/docker.marko [hunk 5]
```diff
@@ -1,62 +1,62 @@
...
+    <p>
+      To install and use these Docker images, you'll need to keep the above in mind while following their documentation. There are many ways to manage Docker images and containers too, so installation
+      and maintenance of them will depend on the route you choose.
+      <ul>
+        <li>
+          <h5>
+            <a href="https://ghcr.io/hotio/sonarr:latest">ghcr.io/hotio/sonarr:release</a>
+          </h5>
+          <p>
+            <a href="https://hotio.dev">hotio</a> doesn't specify any default volumes, besides <code>/config</code>. Images are automatically updated 2x in 1 hour if upstream changes are found. Hotio also builds our Pull Requests which may be useful for testing.
```
Comment (Taloth) [inline src/sections/downloads-v3/docker.marko]: > Hotio also builds our Pull Requests which may be useful for testing.

Are you sure?

File: src/sections/downloads-v3/docker.marko [hunk 6]
```diff
@@ -1,62 +1,62 @@
...
-    </install-step>
-</dl>
+  <install-step title="Introduction">
+    <p>
+      The Sonarr team does not offer an official Docker image. However, a number of third parties have created and maintain their own.<br/>
+      These instructions provide generic guidance that should apply to any Sonarr Docker image.
+    </p>
+    <migration-alert>
+      Most docker containers use <code>/config</code> volume to mount the data directory and supply that path to Sonarr as parameter.
+      Sonarr v3 will convert the given directory on startup if a Sonarr v2 database is found.
```
Comment (Taloth) [inline src/sections/downloads-v3/docker.marko]: Why remove the recommendation to make a backup?

File: src/sections/downloads-v3/windows.marko (hunk 1 above)
Comment (bakerboy448) [reply src/sections/downloads-v3/windows.marko]: Future Proof.  and XP and 7 are covered by this statement.

File: src/sections/support.marko (hunk 3 above)
Comment (bakerboy448) [reply src/sections/support.marko]: yes; reverted

File: src/sections/downloads-v3/docker.marko (hunk 4 above)
Comment (bakerboy448) [reply src/sections/downloads-v3/docker.marko]: oversight

**Comment by bakerboy448** [comment]
@Taloth what (if any) changes are needed to get this landed?

**Pull request closed without merging**